src/NzbDrone.Core/Indexers/Definitions/Avistaz/AvistazParser.cs

```diff
@@ -14,6 +14,9 @@ public class AvistazParser : IParseIndexerResponse
     {
         private readonly HashSet<string> _hdResolutions = new HashSet<string> { "1080p", "1080i", "720p" };
 
+        // Servers are Eastern US Time Zone
+        private readonly TimeZoneInfo _avistazTimeZone = TimeZoneInfo.FindSystemTimeZoneById("US Eastern Standard Time");
```

Comment:
```suggestion
        private readonly TimeZoneInfo _avistazTimeZone = TimeZoneInfo.FindSystemTimeZoneById("America/New_York");
```

Use official IANA timezone to ensure this works on more OSs